### ISSUE TITLE
[BUG] 403 Forbidden (Roblox not loading)

### ISSUE THREAD

**Issue by byeoon** [pizzaboxer/bloxstrap#1988]
### Acknowledgement of preliminary instructions

- [X] I have read the preliminary instructions, and I am certain that my problem has not already been addressed.

### What problem did you encounter?

I'm unable to join any games because whenever I attempt to load into one I get this error `System.Net.Http.HttpRequestException: Response status code does not indicate success: 403 (Forbidden).`

With 2 seconds of research it seems to be happening because of this URL: https://setup.rbxcdn.com/channel/z2waflag/version-2cca5ed32b534b2a-rbxPkgManifest.txt (Maybe there isn't a proper catch incase any of these fail to get a response?)

```
2024-06-14T20:43:10Z [App::OnStartup] Starting Bloxstrap v2.5.4
2024-06-14T20:43:10Z [App::OnStartup] Compiled Friday, 3 November 2023 at 5:46:02 PM from commit 880c076fcd875fc10b5f6bef770033074d79a7a8 (tag/v2.5.4)
2024-06-14T20:43:10Z [App::OnStartup] Loaded from %UserProfile%\AppData\Local\Bloxstrap\Bloxstrap.exe
2024-06-14T20:43:10Z [Logger::Initialize] Initializing at %UserProfile%\AppData\Local\Bloxstrap\Logs\Bloxstrap_20240614T204310Z.log
2024-06-14T20:43:10Z [Logger::Initialize] Finished initializing!
2024-06-14T20:43:10Z [JsonManager<Settings>::Load] Loading from %UserProfile%\AppData\Local\Bloxstrap\Settings.json...
2024-06-14T20:43:10Z [JsonManager<Settings>::Load] Loaded successfully!
2024-06-14T20:43:10Z [JsonManager<State>::Load] Loading from %UserProfile%\AppData\Local\Bloxstrap\State.json...
2024-06-14T20:43:10Z [JsonManager<State>::Load] Loaded successfully!
2024-06-14T20:43:10Z [JsonManager<Dictionary`2>::Load] Loading from %UserProfile%\AppData\Local\Bloxstrap\Modifications\ClientSettings\ClientAppSettings.json...
2024-06-14T20:43:10Z [JsonManager<Dictionary`2>::Load] Loaded successfully!
2024-06-14T20:43:10Z [NotifyIconWrapper::NotifyIconWrapper] Initializing notification area icon
2024-06-14T20:43:10Z [JsonManager<State>::Save] Save request ignored
2024-06-14T20:43:10Z [App::OnStartup] Initializing bootstrapper
2024-06-14T20:43:10Z [App::OnStartup] Initializing bootstrapper dialog
2024-06-14T20:43:10Z [App::OnStartup] Creating singleton mutex
2024-06-14T20:43:10Z [Bootstrapper::Run] Running bootstrapper
2024-06-14T20:43:10Z [Bootstrapper::Run] Performing connectivity check...
2024-06-14T20:43:10Z [Bootstrapper::SetStatus] Connecting to Roblox...
2024-06-14T20:43:10Z [RobloxDeployment::GetInfo] Getting deploy info for channel LIVE (extraInformation=False)
2024-06-14T20:43:10Z [HttpClientLoggingHandler::ProcessRequest] GET https://clientsettingscdn.roblox.com/v2/client-version/WindowsPlayer/channel/LIVE
2024-06-14T20:43:10Z [HttpClientLoggingHandler::ProcessResponse] 200 OK https://clientsettingscdn.roblox.com/v2/client-version/WindowsPlayer/channel/LIVE
2024-06-14T20:43:10Z [Bootstrapper::Run] Connectivity check finished
2024-06-14T20:43:10Z [Bootstrapper::CheckForUpdates] Checking for updates...
2024-06-14T20:43:10Z [HttpClientLoggingHandler::ProcessRequest] GET https://api.github.com/repos/pizzaboxer/bloxstrap/releases/latest
2024-06-14T20:43:10Z [HttpClientLoggingHandler::ProcessResponse] 200 OK https://api.github.com/repos/pizzaboxer/bloxstrap/releases/latest
2024-06-14T20:43:10Z [Bootstrapper::CheckForUpdates] No updates found
2024-06-14T20:43:10Z [RobloxDeployment::GetInfo] Getting deploy info for channel z2waflag (extraInformation=False)
2024-06-14T20:43:10Z [HttpClientLoggingHandler::ProcessRequest] GET https://clientsettingscdn.roblox.com/v2/client-version/WindowsPlayer/channel/z2waflag
2024-06-14T20:43:10Z [HttpClientLoggingHandler::ProcessResponse] 200 OK https://clientsettingscdn.roblox.com/v2/client-version/WindowsPlayer/channel/z2waflag
2024-06-14T20:43:10Z [RobloxDeployment::GetInfo] Getting deploy info for channel LIVE (extraInformation=False)
2024-06-14T20:43:10Z [RobloxDeployment::GetInfo] Deploy information is cached
2024-06-14T20:43:10Z [DeployManager::DefaultBaseUrl.Set] Testing connection to 'https://setup.rbxcdn.com'...
2024-06-14T20:43:10Z [HttpClientLoggingHandler::ProcessRequest] GET https://setup.rbxcdn.com/version
2024-06-14T20:43:11Z [HttpClientLoggingHandler::ProcessResponse] 200 OK https://setup.rbxcdn.com/version
2024-06-14T20:43:11Z [DeployManager::DefaultBaseUrl.Set] Connection successful!
2024-06-14T20:43:11Z [HttpClientLoggingHandler::ProcessRequest] GET https://setup.rbxcdn.com/channel/z2waflag/version-2cca5ed32b534b2a-rbxPkgManifest.txt
2024-06-14T20:43:11Z [HttpClientLoggingHandler::ProcessResponse] 403 Forbidden https://setup.rbxcdn.com/channel/z2waflag/version-2cca5ed32b534b2a-rbxPkgManifest.txt
2024-06-14T20:43:11Z [App::OnStartup] Bootstrapper task has finished
2024-06-14T20:43:11Z [NotifyIconWrapper::Dispose] Disposing NotifyIcon
2024-06-14T20:43:11Z [App::OnStartup] An exception occurred when running the bootstrapper
2024-06-14T20:43:11Z [App::OnStartup] System.AggregateException: One or more errors occurred. (Response status code does not indicate success: 403 (Forbidden).)
 ---> System.Net.Http.HttpRequestException: Response status code does not indicate success: 403 (Forbidden).
   at System.Net.Http.HttpResponseMessage.EnsureSuccessStatusCode()
   at System.Net.Http.HttpClient.GetStringAsyncCore(HttpRequestMessage request, CancellationToken cancellationToken)
   at Bloxstrap.Models.Manifest.PackageManifest.Get(String versionGuid)
   at Bloxstrap.Bootstrapper.CheckLatestVersion()
   at Bloxstrap.Bootstrapper.Run()
   at Bloxstrap.Bootstrapper.Run()
   at Bloxstrap.App.<>c__DisplayClass56_0.<<OnStartup>b__1>d.MoveNext()
   --- End of inner exception stack trace ---
```

**Comment by AmateurProgrammerishere** [comment]
pizzaboxer must fix this i couldnt fix this no matter what i uninstalled temp files and prefecth reinstalled roblox and nothing.

**Comment by pizzaboxer** [comment]
https://github.com/pizzaboxer/bloxstrap/wiki/403-(Forbidden)-error-when-launching-Roblox

**Comment by AmateurProgrammerishere** [comment]
oh i am getting exited for the new update

**Comment by bluepilledgreat** [comment]
Fixed with v2.6.0
